waspc/cli/src/Wasp/Cli/Command/Start/Db.hs

```diff
@@ -107,31 +107,24 @@ startPostgreDevDb waspProjectDir appName = do
   --   only when initializing the database -> if it already exists, they will be ignored.
   --   This is how the postgres Docker image works.
   let command =
-        printf
-          ( unwords
-              [ "docker run",
-                "--name %s",
-                "--rm",
-                "--publish %d:5432",
-                "-v %s:/var/lib/postgresql/data",
-                "--env POSTGRES_PASSWORD=%s",
-                "--env POSTGRES_USER=%s",
-                "--env POSTGRES_DB=%s",
-                "postgres"
-              ]
-          )
-          dockerContainerName
-          Dev.Postgres.defaultDevPort
-          dockerVolumeName
-          Dev.Postgres.defaultDevPass
-          Dev.Postgres.defaultDevUser
-          dbName
+        unwords
+          [ "docker run",
+            printf "--name %s" dockerContainerName,
+            "--rm",
+            printf "--publish %d:5432" Dev.Postgres.defaultDevPort,
+            printf "-v %s:%s" dockerVolumeName postgresDockerVolumePath,
+            printf "--env POSTGRES_PASSWORD=%s" Dev.Postgres.defaultDevPass,
+            printf "--env POSTGRES_USER=%s" Dev.Postgres.defaultDevUser,
+            printf "--env POSTGRES_DB=%s" dbName,
+            postgresDockerImage
+          ]
```

Comment:
Before it was following the idea of "first goes the string tempalte with placeholders, then come the variables", so printf style.
Now, you went it mixing them all together, which someone might find better or not, I see pros and cons, but what you also did is used `printf` isnide for each one, I am not sure you got much with it? Why not just drop prints then and do normal concatenation? Ok there is one line wher eyou have two of htem. But I thikn `printf` here might be more noise than benefit.

But ok I see this is nuanced, so I am not sure -> let's put it this way, I am challenging this a bit, but I am ok with you leaving it like this if you still after that think this is the best.

RAW